crawl-ref/source/beam.cc

```diff
@@ -2369,9 +2356,26 @@ void bolt::affect_endpoint()
             noisy(spell_effect_noise(SPELL_PRIMAL_WAVE),
                   pos(), "You hear a splash.");
         }
-        create_feat_splash(pos(), 2, random_range(3, 12, 2));
+        const int num = agent()->is_player() ? div_rand_round(ench_power * 3, 20) + 3 + random2(7)
```

Comment:
`agent()` can be null, since Fedhas uses primal wave for wrath.

```suggestion
        const int num = agent() && agent()->is_player() ? div_rand_round(ench_power * 3, 20) + 3 + random2(7)
```